grid/lib/utils.py

```diff
@@ -52,16 +51,6 @@ def save_adapter(addr):
     adapter_bin = get_ipfs_api().cat(addr)
     ensure_exists(f'{Path.home()}/grid/adapters/adapter.py', adapter_bin)
 
-
-def keras2ipfs(model):
```

Comment:
Technically, anything that doesn't require a function/class from either of the keras/torch modules could stay, including these functions.  But I think it's a good idea to move these to keras_utils as well, since we really won't need them if we're not loading keras in.  Any other file that uses them will need to import them from there -- that will require some other changes, please see comment in PR about that.